src/main/java/core/basesyntax/service/FruitList.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface FruitList {
+    List<FruitTransaction> getFruitList(List<FruitTransaction> list);
```

Comment:
The method name `getFruitList` is too vague and does not clearly describe the action it performs. A more descriptive name would be better, such as `processTransactions` or `updateFruitQuantities`, depending on what the implementation is intended to do.